include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qr_ks_vs_async.hpp

```diff
@@ -759,6 +759,14 @@ struct BlockFmhaPipelineQRKSVSAsync
                void* smem_ptr,
                DropoutType& dropout) const
     {
+
+        // may have state inside
+        auto pre_softmax_def = PreSoftmaxFunction_{};
```

Comment:
one issue here is, we will get the custom function called only if operator() is called with a shortened signature. I think we should get the custom function called in the exactly opposite case, when it's supplied as an input argument to operator()